nasa/cFE
# Fix #383 - Resolve Explicit Link

**Describe the contribution**
Fix #383, resolving explicit link and any remaining misc. warning. 

**Testing performed**
Steps taken to test the contribution:
1. make usersguide
2. Verify warning is gone

**System(s) tested on:**
 - Hardware
 - Ubuntu 18.04
 - rc-6.7-0

**Contributor Info**
Anh Van, NASA Goddard
